src/util/util_drs.h

```diff
@@ -15,9 +15,11 @@ namespace dxvk::drs {
         ENTRY(NGX_DLSS_OVERRIDE_OPTIMAL_SETTINGS),
         ENTRY(NGX_DLSS_RR_OVERRIDE),
         ENTRY(NGX_DLSS_RR_OVERRIDE_RENDER_PRESET_SELECTION),
+        ENTRY(NGX_DLSS_RR_OVERRIDE_SCALING_RATIO),
         ENTRY(NGX_DLSS_SR_MODE),
         ENTRY(NGX_DLSS_SR_OVERRIDE),
         ENTRY(NGX_DLSS_SR_OVERRIDE_RENDER_PRESET_SELECTION),
+        ENTRY(NGX_DLSS_SR_OVERRIDE_SCALING_RATIO),
```

Comment:
Perhaps we could also add recognition of predefined enum values for new settings? They are `MIN`, `MAX` and `DEFAULT`, although `DEFAULT` being outside of range `MIN` to `MAX` is a bit strange so it probably means "no override".

Reply:
Yep, done